org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/DashboardController.java

```diff
@@ -436,6 +436,15 @@ public  boolean render(Component content, MDashboardContent dc, DashboardRunnabl
 				btn.setLabel(Msg.getMsg(Env.getCtx(), "ViewReportInNewTab"));
 				toolbar.appendChild(new Separator("vertical"));
 				toolbar.appendChild(btn);
+
+				btn = new ToolBarButton();
+				btn.setAttribute("Refresh", true);
+				btn.setAttribute("AD_Process_ID", AD_Process_ID);
+				btn.setAttribute("ProcessParameters", processParameters);
+				btn.setAttribute("Parent", content);
+				btn.setImage(ThemeManager.getThemeResource("images/Refresh16.png"));
```

Comment:
need to cater for css font icon here, i.e if (ThemeManager.isUseFontIconForImage()) btn.setIconSclass(...) else btn.setImage(...). There are plenty of examples in the code.

Reply:
done